earthpy/spatial.py

```diff
@@ -346,8 +362,8 @@ def plot_bands(arr, title=None, cmap="Greys_r", figsize=(12, 12), cols=3, extent
         # Test if there are enough titles to create plots
         if title:
             if not (len(title) == arr.shape[0]):
-                raise ValueError("The number of plot titles should be the same " +
-                                 "as the number of raster layers in your array.")
+                raise ValueError("The number of plot titles should equal " +
+                                 "as the number of array raster layers.")
```

Comment:
Delete "as"? Otherwise we have "The number of plot titles should equal as the number of array raster layers".

Reply:
fixed and also made this a """double line string here"""